packages/app/src/modules/theme.js

```diff
@@ -25,12 +25,29 @@ export const theme = {
                 transition: null,
             },
         },
-        MuiDialogContent: {
-            root: {
-                ...dhis2theme.overrides.MuiDialogContent.root,
-                overflow: 'hidden',
-                overflowY: 'hidden',
+        MuiSelect: {
+            select: {
+                '&:focus': {
+                    background: '$labelcolor',
```

Comment:
Use `background-color` instead of `background`.  What is `$labelcolor` here?  Jan had suggested `transparent` for the background-color.